.github/workflows/sdk.yml

```diff
@@ -24,7 +24,36 @@ jobs:
           working-directory: sdk
           skip-go-installation: true
 
-  test:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1
+    - name: Build SDK
+      working-directory: sdk
+      run: go build ./...
+    - name: Build SDK Tests
+      working-directory: sdk
+      run: go test -exec=echo ./...
```

Comment:
I'm not 100% on how these configs work. Should we do `go test -v` here to see logs of the unit tests?

Reply:
This line with the `-exec=echo` is a hack because there is no way with the Go tooling to tell it to build all the tests in a module without running them. So instead I'm running the tests, but telling it to run the `echo` command instead of the actual tests.

There's an issue open about this limitation here: https://github.com/golang/go/issues/15513.